### PR TITLE
template: convert to SCLogDebug

### PR DESCRIPTION
**Pull request by catenacyber**
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:
https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=44427

Describe changes:
- template: convert to SCLogDebug

Is there a reason it was `SCLogNotice` ?

### CODE REVIEW

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 6856

**Comment by jasonish** [comment]
> Is there a reason it was `SCLogNotice` ?

Mostly to aid in development, it helps to be a little more verbose during initial development. I think this partly comes from the last dev training as well, it helps trace the execution while getting started.

File: src/app-layer-template.c [hunk 1]
```diff
@@ -490,7 +486,7 @@ void RegisterTemplateParsers(void)
 
     if (AppLayerParserConfParserEnabled("tcp", proto_name)) {
 
-        SCLogNotice("Registering Template protocol parser.");
+        SCLogDebug("Registering Template protocol parser.");
```
Comment (jasonish) [inline src/app-layer-template.c:489]: I think this one should at least stay.

**Review by jasonish — CHANGES_REQUESTED** [review]
I think its help to at least leave the registration one during development, so you know all is good.  And in that case we should probably keep the related TODO around.

**Comment by catenacyber** [comment]
Thanks Jason, continued in https://github.com/OISF/suricata/pull/7237

**Pull request closed without merging**